src/Orleans/IDs/ActivationAddress.cs

```diff
@@ -32,6 +33,7 @@ internal class ActivationAddress
         public GrainId Grain { get; private set; }
         public ActivationId Activation { get; private set; }
         public SiloAddress Silo { get; private set; }
+        public MultiClusterStatus Status { get; private set; }
```

Comment:
Good point. I am not quite sure. I think it is only used locally, but I will take a look.